data/ninja-infosys.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Ninja Infosys Pvt. Ltd.",
+  "establishedAt": 1572459300,
+  "location": "Nepalgunj, Banke",
+  "website": "https://ninjainfosys.com/",
+  "email": "ninjainfosys@gmail.com",
+  "phone": "+977-9858042433",
```

Comment:
Does the company have a landline number?
Landline numbers would be much preferred, if not then also it's alright

Reply:
I just updated landline number, please check.